conclusion/conclusion.md

```diff
@@ -0,0 +1,24 @@
+#Conclusion
+
+The team is satisfied with the way the Domain Driven Design approach has been carried on. 
+During the first part of the project the team members gathered to conduct knowledge crunching sessions. 
+These latter allowed the team to reach a uniform knowledge about the domain.
+This was fundamental for the further steps of the project.
+In fact, even though the team members worked on different components of the system, everyone was able to stay coherent and adhere to the project's goal.
+
+Moreover, the definition of subdomains and bounded contexts allowed designing correct size microservices.
+Indeed, they have a well-defined goal and a coherent set of functionalities.
+Also, the classification of the subdomains as _core_, _supporting_ or _generic_ made it easier to assign priorities regarding the implementation of the components of the system.
+For instance, the _authentication microservice_, that is part of the _user subdomain_, has not yet been implemented in order to focus on the most important ones.

```

Comment:
toglierei questa frase

Reply:
mi riferisco a "For instance, the _authentication microservice_, that is part of the _user subdomain_, has not yet been implemented in order to focus on the most important ones."